homeassistant/components/heatmiser/climate.py

```diff
@@ -84,18 +84,13 @@ def __init__(self, therm, device, uh1):
         self._id = device
         self.dcb = None
         self._attr_hvac_mode = HVACMode.HEAT
-        self._temperature_unit = None
+        self._attr_temperature_unit = None
```

Comment:
`temperature_unit` should never be `None`, so I think it was a bug.
Should we instead use a default?

Reply:
It updates before being added to hass, so we should just be able to remove this line completely.

Reply:
```suggestion
```